spec/VehicleSignalSpecification.vspec

```diff
@@ -184,10 +184,10 @@ Vehicle.Exterior:
 #include Vehicle/Exterior.vspec Vehicle.Exterior
 
 #
-# Service information
+# Connectivity information
```

Comment:
The description for this commit is that Vehicle.Connectivity is being added. However, this change seems to be replacing Vehicle.Service with Vehicle.Connectivity. Will Vehicle.Service and its components (such as DistanceToService and TimeToService) no longer be supported?

As well, Connectivity seems to be very specific to ability to connect. Will this still be applicable once further signals are added, such as Wi-Fi SSID, Cellular IMEI, etc.?